<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
    <node pkg="ros_opencl_tests" type="gpu_side_without_ros_opencl" respawn="false" name="ros_opencl_tests_gpu_without_ros_opencl" output="screen">
        <rosparam file="$(find ros_opencl_tests)/config/gpu_without_ros_opencl.yaml" command="load" />
    </node>
</launch>